msfdb

```diff
@@ -1006,6 +968,10 @@ def invoke_command(commands, component, command)
   end
 end
 
+def installed(cmd)
+  !Msf::Util::Helper.which(cmd).nil?
+end
+
 def has_requirements
```

Comment:
Is this functionality dropped now? 👀 

Looks like this had some good logging information when required tools were missing

Reply:
I figured it was kind of superseded by `installed` and different DB setups need different thigns installed but I can probably add something similar tailored to each setup in without too much difficulty

Reply:
I think I might be stupid :sweat_smile: 
it's still being used for the `pg_ctl` path but not `pg_ctlcluster` not sure why I did that, for sure can make it work for either